plugins/flashrom/fu-flashrom-plugin.c

```diff
@@ -291,12 +291,76 @@ fu_flashrom_plugin_find_guid(FuPlugin *plugin, GError **error)
 	return NULL;
 }
 
+typedef char *flashrom_data;
+G_DEFINE_AUTOPTR_CLEANUP_FUNC(flashrom_data, flashrom_data_free)
+
+static gboolean
+fu_flashrom_plugin_probe(FuFlashromPlugin *self, GError **error)
+{
+	gint rc;
```

Comment:
Rc should be in define.

Reply:
both halves of the ifdef need rc -- I can duplicate it if you think it's clearer